### PR TITLE
Implement Telnyx SMS adapter

### LINKED ISSUE #13: 🚀 Feature: Telnyx SMS
### 🔖 Feature description

Support Telnyx (https://developers.telnyx.com/docs/v2/messaging/quickstarts/sending-sms-and-mms) for sending SMS.

### 🎤 Pitch

I'd like Telnyx to be supported.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/utopia-php/messaging/blob/HEAD/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by dcowan-london**
## What does this PR do?

Implements a new SMS Adapter for sending via Telnyx

## Test Plan

I've successfully sent a test SMS using the new Telnyx adapter.

## Related PRs and Issues

Closes #13 

### Have you read the [Contributing Guidelines on issues](https://github.com/utopia-php/messaging/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by stnguyen90** [comment]
@dcowan-london, would you please fix the linting error?

File: src/Utopia/Messaging/Adapters/SMS/Telnyx.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+class Telnyx extends SMSAdapter
+{
+    /**
+     * @param  string  $accountSid Twilio Account SID
+     * @param  string  $authToken Twilio Auth Token
+     */
```
Comment (abnegate) [inline src/Utopia/Messaging/Adapters/SMS/Telnyx.php:13]: Please update these to match the constructor params

**Review by abnegate — COMMENTED** [review]
Looks good!

File: src/Utopia/Messaging/Adapters/SMS/Telnyx.php (hunk 1 above)
Comment (dcowan-london) [reply src/Utopia/Messaging/Adapters/SMS/Telnyx.php:13]: > You can run `composer format` to fix the linting error

Thanks!

> Please update these to match the constructor params
 
Whoops, not sure how I missed those.

**Comment by dcowan-london** [comment]
@stnguyen90 @abnegate Sorry to nag, but following up on this?

**Review by abnegate — APPROVED** [review]
Looks good, thanks for your contribution 😁

**Comment by dcowan-london** [comment]
Thank you!

**Pull request merged**